server/sonar-server/src/test/java/org/sonar/server/computation/queue/CeQueueInitializerTest.java

```diff
@@ -53,30 +45,9 @@
   public TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TemporaryFolder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2924)